hydra/@context/video.jsonld

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": {
+    "@vocab": "http://videos.restfest.org/",
+    "@base": "http://purl.org/restfest/videos/",
+    "sch": "http://schema.org/",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "uri": {
+      "@type": "@id"
+    },
```

Comment:
This should be `"uri": "@id"`. See https://www.w3.org/TR/json-ld/#h3_aliasing-keywords

Reply:
This is not aliasing, this is https://www.w3.org/TR/json-ld/#type-coercion